cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# this script creates all the users/keys on the external cluster
+# those keys will be injected via the import-external-cluster.sh once this one is done running
+# so you can run import-external-cluster.sh right after this script
+set -Eeuo pipefail
+
+#############
+# VARIABLES #
+#############
+
+: "${CLIENT_CHECKER_NAME:=client.healthchecker}"
+
+#############
+# FUNCTIONS #
+#############
+
+function is_available {
+  command -v "$@" &>/dev/null
+}
+
+function checkEnv() {
+  if ! is_available ceph; then
+    echo "'ceph' binary is expected'"
+    exit 1
+  fi
+
+  if ! ceph -s 1>/dev/null; then
+    echo "cannot connect to the ceph cluster"
+    exit 1
+fi
+}
+
+function createCheckerKey() {
+  checkerKey=$(ceph auth get-or-create "$CLIENT_CHECKER_NAME" mon 'allow r, allow command quorum_status'|awk '/key =/ { print $3}')
+  echo "export ROOK_EXTERNAL_CHECKER_SECRET=$checkerKey"
```

Comment:
How about renaming the env vars to something more generic? For example `ROOK_EXTERNAL_USER_SECRET` and `ROOK_EXTERNAL_USERNAME`